sonic_platform_base/module_base.py

```diff
@@ -77,6 +89,83 @@ def get_system_eeprom_info(self):
         """
         raise NotImplementedError
 
+    def get_name(self):
```

Comment:
is this get my name?

Reply:
This will return one of the fixed format names i.e. SUPERVISOR0, LINE-CARD1 or FABRIC-CARD3 for example.